lib/cache-utils.js

```diff
@@ -67,12 +67,10 @@ export async function getCacheKey(scarbLockPath) {
   return `scarb-cache-${platform}-${fileHash}`.toLowerCase();
 }
 
-async function getScarbLockPath(scarbLockPath) {
-  const lockfilePath = scarbLockPath || "Scarb.lock";
-
-  await fs.access(lockfilePath).catch((_) => {
-    throw new Error("failed to find Scarb.lock");
+async function getScarbLockPath(scarbLockPath = "Scarb.lock") {
```

Comment:
this code doesn't cover this case:

```js
"" || "foo" == "foo"
```

which we actually need, as it is pretty easy to declare the input but not set it in yaml